src/segment-loader.js

```diff
@@ -2067,26 +2067,31 @@ export default class SegmentLoader extends videojs.EventTarget {
     });
   }
 
-  handleVideoSegmentTimingInfo_(requestId, videoSegmentTimingInfo) {
+  handleSegmentTimingInfo_(type, requestId, segmentTimingInfo) {
```

Comment:
It took me a second to realize that `type` got passed in here as the first argument. Was wondering how it was being used below 😆